Example/Example/DeviceCheckingViewController.swift

```diff
@@ -17,7 +17,7 @@ class DeviceCheckingViewController: UIViewController {
         if WsneakersUISDKSession.isDeviceSupported() {
             let storyboard = UIStoryboard(name: "Main", bundle: nil)
```

Comment:
DRY: Could you please reuse existed code. The only difference is:
`type: .sneaker`
`type: .watch`
`type: .clothes`